tests/sim/fungible_conversion_proxy.rs

```diff
@@ -0,0 +1,420 @@
+use crate::utils::*;
+use fungible_conversion_proxy::FungibleConversionProxyContract;
+use mocks::fpo_oracle_mock::FPOContractContract;
+use mocks::fungible_token_mock::FungibleTokenContractContract;
+use near_sdk::json_types::{U128};
+use near_sdk_sim::init_simulator;
+use near_sdk_sim::runtime::GenesisConfig;
+use near_sdk_sim::ContractAccount;
+use near_sdk_sim::UserAccount;
+use near_sdk_sim::{call, deploy, lazy_static_include, to_yocto};
+use std::convert::TryInto;
+use std::str;
+
+near_sdk::setup_alloc!();
+
+const FUNGIBLE_PROXY_ID: &str = "fungible_request_proxy";
+lazy_static_include::lazy_static_include_bytes! {
+   FUNGIBLE_REQUEST_PROXY_BYTES => "out/fungible_conversion_proxy.wasm"
+}
```

Comment:
Now that the tests are split up, I think these names could be simplified.

```suggestion
const PROXY_ID: &str = "fungible_conversion_proxy";
lazy_static_include::lazy_static_include_bytes! {
   PROXY_BYTES => "out/fungible_conversion_proxy.wasm"
}
```

Reply:
Thanks, I also adapted the proxy contract variable name accordingly.